custom-namespaces.yaml

```diff
@@ -334,6 +334,7 @@ firefox_accounts:
           table: mozdata.firefox_accounts.events_daily
     growth_accounting:
       type: growth_accounting_view
+      primary_key_field: user_id
```

Comment:
nit: this field isn't a primary key, it has to be combined with the `submission_date` to qualify as that. Maybe call it `identifier_field`?

Reply:
fair enough, renamed the field.